docs/api-docs/cart-and-checkout/embedded-checkout-overview.md

```diff
@@ -27,6 +27,19 @@ You will need to use the [Channels, Sites, and Routes](https://developer.bigcomm
 
 Embedded Checkout requires the BigCommerce Checkout SDK to invoke a method to render the checkout in your site. Learn more about the [Checkout SDK](https://developer.bigcommerce.com/api-docs/cart-and-checkout/checkout-sdk).
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+### Note 
+Translated strings are not supported through Stencil and therefore are not available if using embedded checkout. To display translated strings at checkout, we suggest using a custom [Checkout](https://developer.bigcommerce.com/stencil-docs/localization/multi-language-checkout). 
```

Comment:
This wording has me a little confused. The ticket says "translated strings at the checkout are not supported through Stencil," but if I understand the problem correctly, it's more accurate to say "Strings translated through Stencil are not supported at [embedded] checkout." Maybe you can get some clarification? Or just say something like "Stencil translation files are not supported by embedded checkout" since we know that is the case.

Also, the link to https://developer.bigcommerce.com/stencil-docs/localization/multi-language-checkout has the material you'd need to set up Stencil's multi-language checkout using translation files, but those translation files aren't supported by embedded checkout. Since he mentioned custom checkout, maybe it was supposed to be this link: https://developer.bigcommerce.com/stencil-docs/customizing-checkout/installing-custom-checkouts